package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@wegolook/joi-objectid",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
I vaguely recall previously discussing a schedule for releasing 1.0. 

Were we just waiting on Joi 9.x.x for the `Joi.extend()` functionality, or was there something else?